src/lib/operators/table_scan/column_vs_value_table_scan_impl.hpp

```diff
@@ -56,7 +56,7 @@ class ColumnVsValueTableScanImpl : public AbstractDereferencedColumnTableScanImp
   bool _value_matches_none(const BaseDictionarySegment& segment, const ValueID search_value_id) const;
 
   template <typename Functor>
-  void _with_operator_for_dict_segment_scan(const PredicateCondition predicate_condition, const Functor& func) const {
+  void _with_operator_for_dict_segment_scan(const Functor& func) const {
     switch (predicate_condition) {
```

Comment:
Where does it get the `predicate_condition` from?

Reply:
The class

Reply:
Never mind, instead of column_vs_value I accidentally opened column_vs_column on my Mac and the latter does have `_ predicate_condition`.